generator/testdata/google/example/library/v1/library_service.gapic.h.baseline

```diff
@@ -63,8 +63,27 @@ class LibraryService final {
   }
 
   std::shared_ptr<LibraryServiceStub> stub_;
-  std::unique_ptr<gax::RetryPolicy> retry_policy_;
-  std::unique_ptr<gax::BackoffPolicy> backoff_policy_;
+  std::unique_ptr<google::gax::RetryPolicy> retry_policy_;
+  std::unique_ptr<google::gax::BackoffPolicy> backoff_policy_;
+
+  // Note: conservatively assume no methods are idempotent.
```

Comment:
We probably don't need this comment on every declaration - just once at the top.